src/main/java/at/hannibal2/skyhanni/config/features/event/PrimalFearSolverConfig.java

```diff
@@ -0,0 +1,20 @@
+package at.hannibal2.skyhanni.config.features.event;
+
+import at.hannibal2.skyhanni.config.FeatureToggle;
+import com.google.gson.annotations.Expose;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorBoolean;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigOption;
+
+public class PrimalFearSolverConfig {
+    @Expose
+    @ConfigOption(name = "Math Primal Fear Solver", desc = "Solver for the Math Primal Fear")
+    @ConfigEditorBoolean
+    @FeatureToggle
+    public boolean mathPrimalFear = false;
```

Comment:
nno reason to repeat the name "primal fear" for the variable name in the class that already contains that name